crates/ruff_workspace/src/options.rs

```diff
@@ -2266,11 +2268,34 @@ impl Pep8NamingOptions {
 #[serde(deny_unknown_fields, rename_all = "kebab-case")]
 #[cfg_attr(feature = "schemars", derive(schemars::JsonSchema))]
 pub struct PycodestyleOptions {
-    /// The maximum [width](http://www.unicode.org/reports/tr11/#Overview) to allow for [`doc-line-too-long`] violations within
+    /// The maximum [width](http://www.unicode.org/reports/tr11/#Overview) to allow for [`line-too-long`](rules/line-too-long.md) violations. By default,
```

Comment:
nit: for consistency with your other PR
```suggestion
    /// The maximum [display width](http://www.unicode.org/reports/tr11/#Overview) to allow for [`line-too-long`](rules/line-too-long.md) violations. By default,
```

Reply:
I updated the base PR to use the simpler `width` term. So it should now be consistent. Thanks for flagging.